packages/core/src/extensions/Blocks/api/blockTypes.ts

```diff
@@ -141,6 +149,13 @@ export type Block<BSchema extends BlockSchema> =
     children: Block<BSchema>[];
   };
 
+export type SpecificBlock<
```

Comment:
Might be worth adding a comment for a bit of clarification (the other types are now also commented on the `custom-blocks-example` branch)